lib/audit/checksum.rb

```diff
@@ -3,12 +3,23 @@
 # Checksum validator code
 class Checksum
 
-  # FIXME: remove rubocop exception once we start filling these methods in
-  # rubocop:disable Lint/UnusedMethodArgument
-  # rubocop:disable Style/EmptyMethod:
-  # TODO: implement this;  we begin with a placeholder
-
-  def self.validate_disk(endpoint_name)
+  def self.validate_disk(endpoint_name, limit=Settings.c2m_sql_limit)
```

Comment:
add comment akin to C2M indicating that we must use .each and not .for_each to preserve ordering??